docs/zh-cn/configurator_step_by_step.md

```diff
@@ -0,0 +1,58 @@
+# QMK 配置器: 入门
+
+本章节描述了如何使用QMK配置器构建出固件文件的过程。
```

Comment:
```suggestion
本章节描述了如何使用QMK配置器构建出固件文件的过程。
```

Reply:
this cannot be applied because github ignores "space-only changes", I may fix this in next pr.

Reply:
I included this erroneously, please ignore